deployment/roles/deploy/templates/scheduled-update-cj.yml.j2

```diff
@@ -0,0 +1,33 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: scheduled-update
+spec:
+  schedule: "11 22 * * *"
```

Comment:
Daily felt the best choice for now, because:
- It feels that updates are not that frequent, so falling behind with a day, should not be that much of an issue (especially considering the current frequency of usage of these repositories). If it is, the job can be triggered manually anytime.
- Due to the above, it doesn't feel right at the moment to bombard the API endpoints with thousands of requests multiple times a day.